context/app/static/js/components/Detail/provenance/NodeElement/style.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const Asterisk = styled.div`
```

Comment:
Maybe add `wrapper` to this name since it does not contain the * itself (or make this a functional component that does somehow)?